LICENSE

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
Would be nice to add it in a dedicated PR.

Reply:
I can make a separate PR, but thought a dedicated commit is sufficient because the change is clearly visible in the git history.